content/en/_index.html

```diff
@@ -173,7 +180,13 @@ <h4 class="section-text-bold">
                             </h4>
                         </div>
                         <p class="section-text">
-
+                            Leveraging oneAPI DPC++, oneAPI DPC++ Library and oneAPI MKL, <a href="https://intelpython.github.io/dpnp">dpnp</a> intends to 
```

Comment:
<a href="https://intelpython.github.io/dpnp">dpnp</a> is a drop-in replacement to NumPy that is built on top of SYCL and the oneAPI performance library, oneMKL. dpnp will provide cross-architecture performance portability to NumPy users without requiring minimal changes to their existing NumPy code.